### ISSUE TITLE
Spacewalk/rhn_check compatibility

### ISSUE THREAD

**Issue by svenbroeker** [CpanelInc/yum-plugin-universal-hooks#1]
Hi,

we want to use this plugin to do actions in pretransaction and posttransaction, but we use Spacewalk to Patch centralized. So the Spacewalk trigger the rhn_check remote but this plugin seems to be not triggert then. Since a long time we use yum-plugin-post-transaction-actions, but we want to do actions in pre transaction. Is there a way you could do it working with Spacewalk/rhn_check?

regards

**Comment by drmuey** [comment]
Hello @svenbroeker,

If spacewalk is calling yum and you want to do things before and after every transaction then all you need to do is put scripts that do what you want in ```/etc/yum/universal-hooks/pretrans/``` and ```/etc/yum/universal-hooks/posttrans/```.

And of course ensure that this plugin is enabled and the caller is not calling yum with arguments to disable it.

HTH!

**Comment by svenbroeker** [comment]
Yes with yum it works like charm, but Spacewalk is not using yum.
They are using a combination of osa-dispatcher,jabberd,osa-deamon(client) and rhnsd/rhn-check.
The post-transaction-plugin seems to be do it different.  My Codingskills are not high enough to check what they to.  I've only traced the rhn_check which is called an i saw the plugin is loaded but the actions are not be called. I tried it in Post and Pre.
